cfme/middleware/deployment.py

```diff
@@ -0,0 +1,12 @@
+from cfme.fixtures import pytest_selenium as sel
+
+
+class Deployment(object):
+
+    def __init__(self, name):
+        self.name = name
+        self.provider = 'Middleware'
```

Comment:
Provider should passed through, no? Otherwise the Deployment automatically supposes that the provider name is Middleware.